app/ui/src/app/customizations/api-connector/api-connector-review/api-connector-review.component.ts

```diff
@@ -15,7 +15,7 @@ export class ApiConnectorReviewComponent {
   @Input() showNextButton: boolean;
   @Input() set apiConnectorData(value: ApiConnectorData) {
     this.validation = value;
-    const actionCountByTags = value.actionsSummary.actionCountByTags;
+    const actionCountByTags = value.actionsSummary.actionCountByTags || {};
```

Comment:
According to some authors, short circuiting property values like this is a code smell, since we eliminate the possibility of taking action whenever the value of `actionCountByTags` is `undefined` or null (or even 0). I tend to agree with this.

I’d rather had wrapped the logic beneath this statement with a more expressive `if(actionCountByTags)` instead, thus allowing us to provide a different logic when no tags were returned.

In any event, I approve this...